gef.py

```diff
@@ -1928,7 +1928,8 @@ def is_branch_taken(self, insn):
         taken, reason = False, ""
 
         if mnemo in {"cbnz", "cbz", "tbnz", "tbz"}:
-            reg = operands[0]
+            # register name is expected to start with $
+            reg = f"${operands[0]}"
```

Comment:
Great catch!

I'd just argue that it'd be better to change `reg` using `str.format` or `%` string formatter simply for compatibility reasons: IIRC f-strings appeared in Py 3.7 and some GDB versions of some old distros (like Ubuntu 16.04) still ships with 3.6.

Also the comment is not necessary.

Reply:
Thank you for review. Fixed.